breacharbiter.go

```diff
@@ -654,29 +747,15 @@ justiceTxBroadcast:
 				revokedFunds += inp.Amount()
 			default:
 			}
-		}
 
-		brarLog.Infof("Justice for ChannelPoint(%v) has "+
-			"been served, %v revoked funds (%v total) "+
-			"have been claimed", breachInfo.chanPoint,
-			revokedFunds, totalFunds)
-
-		err = b.cleanupBreach(&breachInfo.chanPoint)
```

Comment:
Where's `cleanupBreach` called now? The new logic doesn't seem to have a proper termination point.

Reply:
It's still being called the same place, just that the logic before it has been extracted into a method.